src/core/client/stream/test/profile/myComments.spec.tsx

```diff
@@ -115,3 +127,27 @@ it("loads more comments", async () => {
     commentsBefore + 1
   );
 });
+
+it("shows archived notification when archiving enabled", async () => {
+  const testRenderer = createTestRenderer({ archivingEnabled: true });
+  const commentHistory = await waitForElement(() =>
+    within(testRenderer.root).getByTestID("profile-commentHistory")
+  );
+
+  expect(
+    within(commentHistory).getByTestID("archivedComments-thresholdNotification")
+  ).toBeDefined();
```

Comment:
What if we get this by the actual text that we expect it to show to check that's as expected? I also think that would be more readable.

Reply:
Can do, but I won't be able to do the whole text, since it's interpolated, but I'll do the first bit that is consistent and searchable. Should get it towards more read-able.